bazel/repository_locations.bzl

```diff
@@ -1196,7 +1196,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2023-03-03",
         cpe = "cpe:2.3:a:google:flatbuffers:*",
         license = "Apache-2.0",
-        license_url = "https://github.com/google/flatbuffers/blob/v{version}/LICENSE.txt",
```

Comment:
this one has been renamed -> `LICENSE` (including in our current version)

Reply:
That's correct. Please check my commit again - the new name "LICENSE" is used.

Ok? (or maybe I did not understand your comment)

Reply:
my bad - i read it the wrong way round